lib/features/modeling/ElementFactory.js

```diff
@@ -195,11 +195,12 @@ ElementFactory.prototype._getDefaultSize = function(semantic) {
 };
 
 
-ElementFactory.prototype.createParticipantShape = function(collapsed) {
+ElementFactory.prototype.createParticipantShape = function(attrs) {
```

Comment:
Is this always an object / never a boolean?

Reply:
I just checked, sometimes it's a boolean. My bad. Will fix. 🛠